src/main/java/slimeknights/tconstruct/library/data/tinkering/AbstractModifierProvider.java

```diff
@@ -122,7 +122,11 @@ public void run(CachedOutput cache) throws IOException {
   private static JsonObject serializeModifier(@Nullable Modifier modifier, @Nullable ICondition condition, JsonRedirect[] redirects) {
     JsonObject json;
     if (modifier != null) {
-      json = ModifierManager.MODIFIER_LOADERS.serialize(modifier).getAsJsonObject();
+      try {
+        json = ModifierManager.MODIFIER_LOADERS.serialize(modifier).getAsJsonObject();
+      } catch (Exception e) {
+        throw e;
+      }
```

Comment:
What is the reason for this change? This block does nothing effectively

Reply:
Leftover debug I forgot to remove.